src/Dotnet.Script/Program.cs

```diff
@@ -42,6 +43,8 @@ private static int Wain(string[] args)
 
             app.HelpOption("-? | -h | --help");
 
+            app.VersionOption("-v | --version", () => $"{typeof(Program).GetTypeInfo().Assembly.GetName().Version.ToString(3)}-beta");
+
```

Comment:
I think you should use value of AssemblyInformationalVersionAttribute instead of assembly version.

So there will no need to hardcode '-beta' in code, since it is subject to change in future.

Reply:
👍 x 💯

Reply:
this is a good suggestion. Something like this: `typeof(Program).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;` will indeed automatically print `0.13.0-beta` when built with the beta suffix, and `0.13.0` when built without it.

Reply:
Fixing it now :)

Reply:
I've included the VersionSuffix in the csproj file. Makes it easier to read the version info when building choco packages which is what I am doing now  👍